cyclonedx/model/bom.py

```diff
@@ -226,6 +54,7 @@ def from_parser(parser: BaseParser) -> 'Bom':
         """
         bom = Bom()
         bom.components.update(parser.get_components())
+        bom.metadata.update(parser.get_metadata())
```

Comment:
:x:
since this seams to be the only relevant change here, please add proper unit tests